examples/domainmodel/test/refs-index.test.ts

```diff
@@ -62,7 +60,7 @@ function nodeRefString(from: AstNode, to: AstNode): string {
 }
 
 function createPath(node: AstNode): string {
-    return services.workspace.AstNodeLocator.getAstNodePath(node);
+    return services.index.AstNodeLocator.getAstNodePath(node);
```

Comment:
There might've been an issue during rebasing:
```suggestion
    return services.workspace.AstNodeLocator.getAstNodePath(node);
```